docker-compose.yml

```diff
@@ -3,16 +3,19 @@ db:
   environment:
     POSTGRES_USER: postgres
     POSTGRES_PASSWORD: postgres
+cache:
+  image: library/memcached
 web:
-  image: kinto/kinto-server
+  image: ema-kinto-server
   links:
    - db
+   - cache
   ports:
    - "8888:8888"
   environment:
-    KINTO_CACHE_BACKEND: kinto.core.cache.postgresql
-    KINTO_CACHE_URL: postgres://postgres:postgres@db/postgres
+    KINTO_CACHE_BACKEND: kinto.core.cache.memcached
+    KINTO_CACHE_HOSTS: 127.0.0.1:11211 127.0.0.1:11212
```

Comment:
Can we use a DNS to the library/memcached image there?

Reply:
I made the changes accordingly